.travis.yml

```diff
@@ -35,6 +41,24 @@ jobs:
         - export PY_OUT_DIR=py
         - mkdir -p "$PY_OUT_DIR" && python3 -m grpc_tools.protoc -Isdk --python_out=$PY_OUT_DIR --grpc_python_out=$PY_OUT_DIR sdk/*.proto
       name: "Protobuf code generation"
+    - stage: release to github
+      name: linux packages
```

Comment:
Let's use `"` as in every other name, for consistency

Reply:
done, and rebased to be in commit only